### PR TITLE
build(devDependencies): Use devDependencies instead of a custom Gulp task

### PR DESCRIPTION
**Pull request by merlinnot**
Using devDependencies enables more npm utilities to be used, like `npm audit`, `npm outdated`. It
also adds entries to the package-lock.json, which is important for the performance of a CI/CD
systems.

### CODE REVIEW

**Comment by nchanged** [comment]
@merlinnot let's discuss this thingy. We need to get rid of some old tests

File: gulpfile.js [hunk 1]
```diff
@@ -352,46 +352,3 @@ gulp.task("dev", () => {
...
-		"coffee-script",
-		"@types/node",
-		"vue-template-compiler",
-		"vue-template-es2015-compiler",
-		"vue",
-		"jwt-decode",
-		"vue-server-renderer",
-		"vue-hot-reload-api",
-		"vue-class-component",
-		"rollup",
```
Comment (nchanged) [inline gulpfile.js]: Rollup we don't need 100%

File: gulpfile.js [hunk 2]
```diff
@@ -352,46 +352,3 @@ gulp.task("dev", () => {
...
-		"buble",
-		"consolidate",
-		"pug",
-		"tslint",
```
Comment (nchanged) [inline gulpfile.js]: Let's remove tslint for now. We will add it later

File: gulpfile.js [hunk 3]
```diff
@@ -352,46 +352,3 @@ gulp.task("dev", () => {
...
-		"@angular/core",
-		"stylus",
-		"less",
-		"postcss",
-		"postcss-selector-parser",
-		"marked",
-		"node-sass",
-		"uglify-js",
-		"uglify-es",
-		"source-map@0.5.7",
```
Comment (nchanged) [inline gulpfile.js]: Not sure if it's needed here.

File: gulpfile.js [hunk 4]
```diff
@@ -352,46 +352,3 @@ gulp.task("dev", () => {
 		},
 	);
 });
-gulp.task("installDevDeps", function(done) {
-	var deps = [
-		"babel-core",
-		"babel-generator",
```
Comment (nchanged) [inline gulpfile.js]: let's remove babel-core and babel-generator tests. They are useless

File: gulpfile.js [hunk 5]
```diff
@@ -352,46 +352,3 @@ gulp.task("dev", () => {
...
-		"babel-preset-latest",
-		"babel-plugin-transform-es2015-modules-commonjs",
-		"babylon",
-		"cheerio",
-		"@angular/core",
```
Comment (nchanged) [inline gulpfile.js]: Questionable. I think we should remove it too

File: gulpfile.js [hunk 6]
```diff
@@ -352,46 +352,3 @@ gulp.task("dev", () => {
...
-		"node-sass",
-		"uglify-js",
-		"uglify-es",
-		"source-map@0.5.7",
-		"coffee-script",
-		"@types/node",
-		"vue-template-compiler",
-		"vue-template-es2015-compiler",
-		"vue",
-		"jwt-decode",
```
Comment (nchanged) [inline gulpfile.js]: Do we really need this one. Hm. ?!

File: gulpfile.js (hunk 3 above)
Comment (merlinnot) [reply gulpfile.js]: I've updated it and tested. There is a 1.0.0-rc.2 release, I hope it will stabilise soon.

Comment (nchanged) [reply gulpfile.js]: Actually, we don't have tests for it... So you can remove it

Comment (merlinnot) [reply gulpfile.js]: It's a dependency, not a devDependency. Are you sure? I'm fairly certain it should stay as is.

**Comment by merlinnot** [comment]
@nchanged can you take another look?

File: gulpfile.js (hunk 3 above)
Comment (nchanged) [reply gulpfile.js]: oh yeah you are right. it's just not used. It's for those who don't use any transpilers.

**Pull request merged**